### DISCUSSION TITLE
Do jobs resume?

### DISCUSSION THREAD

**Discussion by devlzcode** [taskforcesh/bullmq#2219] [Q&A]
If the worker process unexpectedly closes during a long running job, will that job be automatically resumed once a worker is ready to pick it up? For clarity its better to ask if the job will start again since it was never completed but was active. I think in this case the job is considered stalled.

**Comment by manast** [comment]
TLDR; yes. The mechanism is explained here: https://docs.bullmq.io/guide/jobs/stalled
